README.md

```diff
@@ -21,53 +20,47 @@ Below are instructions to quickly install and configure nginMesh.  Currently, on
 ### Prerequisites
 Make sure you have a Kubernetes cluster with at least 1.9 or greater due to fact only automatic sidecar injection is supported and no alpha feature is enabled. Please see [Prerequisites](https://istio.io/docs/setup/kubernetes/quick-start.html) for setting up a kubernetes cluster.
 
-### Installing Istio and nginMesh
+### Install Istio and nginMesh
 nginMesh requires installation of Istio first.
 
 1. Download and install Istio 0.7.1:
 ```
 curl -L https://git.io/getLatestIstio | ISTIO_VERSION=0.7.1 sh -
 ```
-2. Download nginMesh release 0.7.2:
+2. Download nginMesh release 0.7.1:
 ```
-curl -L https://github.com/nginmesh/nginmesh/releases/download/0.7.2/nginmesh-0.7.2.tar.gz | tar zx
+curl -L https://github.com/nginmesh/nginmesh/releases/download/0.7.1/nginmesh-0.7.1.tar.gz | tar zx
 ```
-
 3. Deploy Istio between sidecars:
-
 ```
 kubectl create -f istio-0.7.1/install/kubernetes/istio.yaml
 ```
-
-
 4. Ensure the following Kubernetes services are deployed: istio-pilot, istio-mixer, istio-ingress:
 ```
 kubectl get svc  -n istio-system  
 ```
 ```
- NAME            CLUSTER-IP      EXTERNAL-IP       PORT(S)                       AGE
-  istio-ingress   10.83.245.171   35.184.245.62     80:32730/TCP,443:30574/TCP    5h
-  istio-pilot     10.83.251.173   <none>            8080/TCP,8081/TCP             5h
-  istio-mixer     10.83.244.253   <none>            9091/TCP,9094/TCP,42422/TCP   5h
+istio-ingress            LoadBalancer   10.47.252.40    35.237.173.47   80:32171/TCP,443:32198/TCP                   19h
+istio-mixer              ClusterIP      10.47.251.225   <none>          9091/TCP,15004/TCP,9093/TCP,9094/TCP,9102/TCP,9125/UDP,42422/TCP    19h
+istio-pilot              ClusterIP      10.47.254.118   <none>          15003/TCP,15005/TCP,15007/TCP,15010/TCP,8080/TCP,9093/TCP,443/TCP   19h
+istio-sidecar-injector   ClusterIP      10.47.242.139   <none>          443/TCP                                       9h
 ```
-
 5. Ensure the following Kubernetes pods are up and running: istio-pilot-* , istio-mixer-* , istio-ingress-*  and istio-initializer-* :
 ```
 kubectl get pods -n istio-system    
 ```
 ```
-  istio-ca-3657790228-j21b9           1/1       Running   0          5h
-  istio-ingress-1842462111-j3vcs      1/1       Running   0          5h
-  istio-pilot-2275554717-93c43        1/1       Running   0          5h
-  istio-mixer-2104784889-20rm8        2/2       Running   0          5h
+istio-ca-86f55cc46f-nprhw                1/1       Running   0          19h
+istio-ingress-5bb556fcbf-c7tgt           1/1       Running   0          19h
+istio-mixer-86f5df6997-fvzjx             3/3       Running   0          19h
+istio-pilot-67d6ddbdf6-xhztz             2/2       Running   0          19h
+istio-sidecar-injector-5b8c78fd6-8dvq6   1/1       Running   0          9h
 ```
-
 6. Automatic sidecar:
```

Comment:
`Automatic sidecar:` -> `Enable automatic sidecar injection`
Remove `To set up sidecar injection, please run following script which will install Istio webhook with nginMesh customization.`

Reply:
ok